src/background/services/wallet/WalletService.ts

```diff
@@ -915,34 +915,32 @@ export class WalletService implements OnUnlock {
       return [];
     }
 
-    assertPresent(indices[0], SecretsError.NoAccountIndex);
```

Comment:
This assertion was too early, this sometimes gets called this without any indices passed